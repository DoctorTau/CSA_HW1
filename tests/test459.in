647
7905 -4342 8168 6475 9644 1424 7495 -6011 -2989 3440 -383 8864 -6583 -9780 8317 5519 -182 -5292 5888 5357 542 -4816 -2383 -6412 1871 -7848 -5593 -217 8518 4973 6571 -4245 -4636 -6219 5749 -9681 -8606 3860 -571 -9319 -6343 -2735 9694 4468 8327 2825 6579 1977 -6010 -5931 -2259 8631 8378 6223 4724 -8652 -1645 2121 6836 2993 -7170 4198 1670 -4799 8396 -4033 3707 -4241 7415 -1099 -3024 -6969 -7609 -8635 -940 -5046 3998 9717 854 -7550 4492 3771 -1597 -7457 5851 663 4643 6218 4001 -2203 382 -1581 -5719 21 -2127 2104 -4976 -2688 9620 1707 -7656 3013 4038 -4134 -6771 6261 -7318 7762 1135 3581 -521 -1785 8723 182 -522 -6383 834 -4073 5241 6609 -4258 6081 -6086 -5389 -6438 5342 -9687 9801 -2032 4378 -6275 -582 -2242 -4302 9150 -2190 -5439 4211 9549 7071 9723 7278 -4779 -3139 2328 9101 4006 4952 6181 -735 4300 6239 565 9911 -5923 -8600 7506 -7927 -235 9635 1506 5823 -3978 5505 8579 4044 -4770 -461 4548 -3371 6508 734 -3486 -8386 -7703 2643 -578 6451 -7243 6108 4676 -2326 -2619 -7294 -197 2014 -1202 -7012 -4178 -2350 683 -642 9679 -5682 -8669 -4540 1700 7318 9799 -5395 -5019 7906 -1327 -2139 2136 2468 7841 -9576 -1454 4653 -2681 1823 1458 -8478 1016 -5845 2335 1308 -99 5802 207 -9006 -9848 1359 -102 -6162 -9623 -1014 -7431 2854 -1097 -926 6422 -6671 6892 -1448 5815 4811 -5946 7202 -1819 -1028 9597 -985 -3148 6557 -5148 1246 -5686 -6822 -8784 6506 388 -7579 -6040 4863 5812 -2968 -1000 -314 1843 -9128 -2018 -2351 -2225 -2259 4348 1514 2885 9973 2719 6395 -7735 992 6455 -9463 -5953 -6829 5986 -2277 -4203 -2124 -6112 -4093 -4849 1246 7291 -3499 -9900 -9617 -7284 -5578 -2166 6065 3646 -5589 -9115 4990 -9106 58 -4748 2182 2630 4293 2836 -7436 -3546 -9120 -3455 -6619 -7446 4903 -8719 9469 -5780 525 -1911 -1448 826 -7113 6010 8672 2587 -1149 -1634 -5495 -488 -9187 -5656 8583 8371 -1372 -2336 6920 657 -7417 -899 6440 -1660 2653 -5290 -696 -3158 -8200 -8920 -230 4040 -9829 -9729 -5198 -5615 8977 9535 7732 8093 8716 5953 -5576 -4886 -3106 5105 8331 5402 -50 7962 7749 -4026 9984 -6709 -3498 -3427 4683 -4480 -3215 -5005 -8439 8775 -9169 7846 -7309 1412 4171 4556 -6568 -5693 925 8502 -7581 6048 203 2415 -7727 6764 5100 -5508 5527 -930 -2166 7163 -1317 -8294 -2760 -6467 -3414 9592 6425 8520 2105 -8353 1956 2846 -2875 -8678 3592 9763 5161 -2514 6248 -6591 -3500 3903 -5239 5826 -4556 -2629 4076 -6358 -5304 -9652 -4387 123 2140 8086 873 -8449 -4434 -9943 1875 -6737 2895 -1471 3150 -3613 -5635 9526 8125 6301 8840 -65 -3171 422 7273 -7511 -8740 2765 -770 -2794 -5979 6122 -9828 4316 -829 5455 8212 -1308 6216 2730 1379 2062 -1355 2643 9816 -2666 7108 9067 1397 -3699 -125 -5481 9799 7495 7691 -4004 4788 6239 -4162 -7820 -1053 8575 2080 1343 -9094 -5895 -2973 -4076 -6677 1141 5315 7044 4007 -9262 3915 428 3399 -4365 6563 -3366 5022 -3308 -1794 -620 -1888 7559 262 -7978 9448 -5824 -6115 6431 7386 -6024 4461 -3184 9874 -1082 -7703 -3077 3266 -2835 7110 -733 2673 -8944 -9537 -3963 3913 -7384 5047 3227 -5173 -9773 5843 -6765 1257 8134 -8330 5638 -6582 8044 4288 2530 162 -6317 1108 -3295 2373 -4319 -6296 4737 4771 1477 -407 -1859 -7671 9646 -9625 -9295 3990 -8071 5580 5103 7490 -99 4103 9658 -532 9201 -7014 4218 -1278 3827 5464 4714 6418 -8331 -8227 5551 9928 6868 -7484 337 -9592 -1882 4433 -6678 -2101 -408 1476 6988 2340 -4128 6946 1453 6810 306 237 4480 5210 8564 -5321 -3270 -8289 9259 -29 -911 9579 6603 -3314 -8131 -8230 -61 -9949 -6443 5883 -9718 -9464 -6954 7663 4003 -3680 5557 -3340 4986 -3622 -7275 -4144 2806 5681 -6815 7619 -4349 1951 -2295 -6584 2111 -9862 -5719 