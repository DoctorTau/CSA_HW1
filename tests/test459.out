-4342 -9780 -182 -5292 -4816 -6412 -7848 -4636 -8606 -6010 -8652 -7170 -3024 -940 -5046 -7550 -4976 -2688 -7656 -4134 -7318 -522 -4258 -6086 -6438 -2032 -582 -2242 -4302 -2190 -8600 -3978 -4770 -3486 -8386 -578 -2326 -7294 -1202 -7012 -4178 -2350 -642 -5682 -4540 -9576 -1454 -8478 -9006 -9848 -102 -6162 -1014 -926 -1448 -5946 -1028 -3148 -5148 -5686 -6822 -8784 -6040 -2968 -1000 -314 -9128 -2018 -2124 -6112 -9900 -7284 -5578 -2166 -9106 -4748 -7436 -3546 -9120 -7446 -5780 -1448 -1634 -488 -5656 -1372 -2336 -1660 -5290 -696 -3158 -8200 -8920 -230 -5198 -5576 -4886 -3106 -50 -4026 -3498 -4480 -6568 -5508 -930 -2166 -8294 -2760 -3414 -8678 -2514 -3500 -4556 -6358 -5304 -9652 -4434 -8740 -770 -2794 -9828 -1308 -2666 -4004 -4162 -7820 -9094 -4076 -9262 -3366 -3308 -1794 -620 -1888 -7978 -5824 -6024 -3184 -1082 -8944 -7384 -8330 -6582 -6296 -532 -7014 -1278 -7484 -9592 -1882 -6678 -408 -4128 -3270 -3314 -8230 -9718 -9464 -6954 -3680 -3340 -3622 -4144 -6584 -9862 